e2e/ansible/playbooks/resiliency/test-disk-failure/test-disk-failure.yml

```diff
@@ -0,0 +1,179 @@
+         changed_when: True
+
+       - name: Get the disk name by using UUID
+         shell: blkid | grep {{ uuid }} | awk '{print $1}' | cut -d "/" -f3 | cut -d ":" -f1
+         args:
+           executable: /bin/bash
+         become: true
+         register: result_disk
+         delegate_to: "{{ groups['kubernetes-kubeminions'].0 }}"
+
+       - name: Get the Mount point of the disk using UUID
+         shell: lsblk -f | grep {{ uuid }} | awk '{print $4}'
+         args:
+           executable: /bin/bash
+         become: true
+         register: result_mountpoint
+         delegate_to: "{{ groups['kubernetes-kubeminions'].0 }}"
+
+       - name: Check status of maya-api server
+         include_tasks: "{{ ansible_env.HOME }}/{{utils_path}}/deploy_check.yml"
+         vars:
+           ns: default
+           app: maya-api
+
+       - name: Get the storage pool creation YAML file
+         get_url:
+           url: "{{ storage_pool_link }}"
+           dest: "{{ result_kube_home.stdout }}"
+         register: result_storagepool
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Replace storage pool name with test parameters
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/regex_task.yml"
+         vars:
+           path: "{{ result_kube_home.stdout }}/storage-pool-path_creation.yml"
+           regex1: "{{replace_item}}"
+           regex2: "{{replace_with}}"
+
+       - name: create the storage pool path
+         shell: source ~/.profile; kubectl apply -f storage-pool-path_creation.yml
+         args:
+           executable: /bin/bash
+         register: result_sp_path
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Obtaining storage classes yaml
+         shell: kubectl get sc openebs-percona -o yaml > "{{ create_sc }}"
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Replace storage class file with test parameters
+         replace:
+           path: "{{ result_kube_home.stdout }}/{{ create_sc }}"
+           regexp: 'openebs.io/storage-pool: default'
+           replace: 'openebs.io/storage-pool: disk'
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name:  Delete the existing storage class and create new one
+         shell: kubectl delete sc openebs-percona; kubectl apply -f "{{ create_sc }}"
+         args:
+           executable: /bin/bash
+         register: result_sc_out
+         until: "'created' in result_sc_out.stdout"
+         delay: 10
+         retries: 5
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Obtain the sql test script file
+         get_url:
+           url: "{{ sql_test_link }}"
+           dest: "{{ result_kube_home.stdout }}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Create the configmap test
```

Comment:
Deploy test application on its own namespace and the namespace name should be test specific and unique.